remote/remote_test.go

```diff
@@ -501,6 +501,40 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 		})
+
+		when("#RetryLogic", func() {
```

Comment:
```suggestion
		when("retry logic", func() {
```
(for other scopes that have `#Thing`, `Thing` is an accessible function from the package)